Sources/JavaKitExample/JavaKitExample.swift

```diff
@@ -109,12 +109,12 @@ extension JavaClass<HelloSwift> {
 
 @JavaClass("com.example.swift.HelloSubclass", extends: HelloSwift.self)
 struct HelloSubclass {
-  @JavaField
+  @JavaStaticField
   var greeting: String
```

Comment:
shouldn't we add another static field rather than modify this one which covers a normal property?

Reply:
Same issue, I put this on the wrong type (but this kind of shows why I think `JavaStaticField` is not needed since this worked when placed on a non static field)